docs/changes/490.misc.rst

```diff
@@ -0,0 +1 @@
+Fix ruff compressions and performance issues, limit new version of sphinx, add Py312,Py313 to trove classifiers
```

Comment:
The sphinx issue has been addressed and the classifiers have been added since this was written. Please could you correct these.